components/Quiz.tsx

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable max-len */
+
+import CorrectIcon from './icons/Correct';
+import IncorrectIcon from './icons/Incorrect';
+import { useState } from 'react';
+
+interface IconWrapperProps {
+  children: React.ReactNode
+}
+export enum QuizType {
+  trueOrFalse = '是非題',
+  quickReview = '快速回顧'
+}
+interface QuizProps {
+  type: QuizType,
+  options: QuizOption[],
+  children: React.ReactNode
+}
+interface QuizHeadingProps {
+  type: QuizType,
+  children: React.ReactNode
+}
+interface QuizOptionProps {
+  option: QuizOption,
+  answer: QuizOption,
+  isSubmit: boolean,
+  isCorrect: boolean,
+  onClick: Function
+}
+interface QuizOption {
+  text: string,
+  isCorrect: boolean
+}
+interface QuizSubmitButtonProps {
+  onClick: Function
+}
+interface QuizAnswerReult {
+  isCorrect: boolean,
+  answer: QuizOption,
+  options: QuizOption[]
+}
+
+const IconWrapper = ({ children }: IconWrapperProps) => {
+  return (
+    <span className="leading-none	mx-2 align-text-bottom">
```

Comment:
```suggestion
    <span className="leading-none mx-2 align-text-bottom">
```

Reply:
I see my code doesn't have this strange space
<img width="376" alt="image" src="https://user-images.githubusercontent.com/80741494/188319116-31fa11ca-6b45-42bb-84ab-59109a2455e2.png">